tests/acceptance/pageObjects/filesPage.js

```diff
@@ -119,6 +120,28 @@ module.exports = {
         .useXpath()
         .waitForElementNotPresent('@loadingIndicator')
         .waitForElementNotPresent(this.getFileRowSelectorByFileName(element))
+    },
+    toggleFilterFileOrFolder: function (fileOrFolder, enableOrDisable) {
+      let labelSelector, checkboxId
+      if (fileOrFolder === 'folder') {
+        labelSelector = '@filterFolderLabel'
+        checkboxId = this.elements['filterFolderCheckbox']
+      } else if (fileOrFolder === 'file') {
+        labelSelector = '@filterFileLabel'
+        checkboxId = this.elements['filterFileCheckbox']
+      } else {
+        throw new Error(`Expected 'file' or 'folder', ${fileOrFolder} given`)
+      }
+      return this
+        .waitForElementVisible('@filterListButton')
+        .click('@filterListButton')
+        .waitForElementVisible(labelSelector, () => {
+          webUIAction.toggleCheckbox(
```

Comment:
we should be able to use
```suggestion
          .waitForElementVisible(labelSelector)
          .webUIAction.toggleCheckbox(enableOrDisable, checkboxId)
```